apps/backend/src/deposits/pos-deposit.service.ts

```diff
@@ -86,7 +92,7 @@ export class PosDepositService {
     );
     return await this.posDepositRepo.find({
       where: {
-        transaction_date: LessThan(date),
+        transaction_date: LessThan(maxDate),
```

Comment:
Can we rename this maxDate to something like "posExceptionsDate", or "pastDueDate", or something along those lines? I think min/max make sense when we're using a range. It would be good to set the variable for posExceptionsDate (or any other name that makes sense) in the lambda or reconciliation service in case we need to make adjustments later on. ?

Reply:
Doing this